assets/src/edit-story/app/prepublish/text.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import stripHTML from '../../utils/stripHTML';
+
+const MAX_PAGE_CHARACTER_COUNT = 200;
+const MIN_STORY_CHARACTER_COUNT = 100;
+
+function characterCountForPage(page) {
+  let characterCount = 0;
+  page.elements.forEach((element) => {
+    if (element.type === 'text') {
+      characterCount += stripHTML(element.content).length;
+    }
+  });
+  return characterCount;
+}
+
+/**
+ * Check page for too much text
+ *
+ * @param  {Object} page Page object
+ * @return {Object} Prepublish check response
+ */
+export function pageTooMuchText(page) {
+  const characterCount = characterCountForPage(page);
+
+  if (characterCount > MAX_PAGE_CHARACTER_COUNT) {
+    return {
+      message: __('Too much text on page', 'web-stories'),
+      pageId: page.id,
+      type: 'guidance',
```

Comment:
We probably should have an enum somewhere. Something like

```
const PRE_PUBLISH_MESSAGE_TYPES = {
  GUIDANCE: 'guidance',
  // ...
}
```

Reply:
Hey Zach, I added some typedefs and this enum on my PR. What do you think about these changes?
https://github.com/google/web-stories-wp/pull/5029/commits/695cc783e1a9ef5e632569ef7c48e21c9bd59026

Reply:
@embarks that all looks great to me. I'll adopt those types and enum after your merge.